packages/babel-generator/test/index.js

```diff
@@ -295,6 +295,15 @@ describe("generation", function() {
     expect(generated).toHaveProperty("map");
     expect(typeof generated.map).toBe("object");
   });
+
+  it("wraps around infer inside an array type", () => {
```

Comment:
Nit, but can we move this down with the other typescript parens tests below?

https://github.com/smelukov/babel/blob/force-parentheses-around-infer/packages/babel-generator/test/index.js#L464